README-en.md

```diff
@@ -0,0 +1,29 @@
+# Serenata de amor
+> Fighting corruptions in a massive and effective way
+
+[Jump to the PT-BR version](README.md)*
```

Comment:
Ow, my bad then! Sorry…

On Mon, Aug 8, 2016 at 2:55 PM flp <notifications@github.com> wrote:

> In README-en.md
> <https://github.com/datasciencebr/serenata-de-amor/pull/3#discussion_r73878746>
> :
>
> > @@ -0,0 +1,29 @@
> > +# Serenata de amor
> > +> Fighting corruptions in a massive and effective way
> > +
> > +[Jump to the PT-BR version](README.md)*
>
> I thought github didn't support 🇧🇷 flag:
> [image: image]
> <https://cloud.githubusercontent.com/assets/2338778/17481736/f8c92f4c-5d54-11e6-93bb-ab80e477a674.png>
>
> *OSX or people with emoji installed probably can see the flag, my bad.*
>
> —
> You are receiving this because you were mentioned.
>
> Reply to this email directly, view it on GitHub
> <https://github.com/datasciencebr/serenata-de-amor/pull/3/files/2cbd4d4f227f334b7c2a9d32bea7b31fc0ad8656#r73878746>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEg383Txw2D6qI6Vf1pcOqVU68lvgEX2ks5qdzVigaJpZM4JWdeZ>
> .
>